packages/react-conformance/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@fluentui/react-conformance",
+  "version": "0.1.0",
+  "description": "Conformance testing",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
```

Comment:
You're missing a devDependency on `jest` (copy the version from another package). You should also add these peerDependencies, which typically use a wider range:
```
"enzyme": "^3.0.0",
"enzyme-adapter-react-16": "^1.0.0",
"jest": "^24.0.0"
```
(these dep things won't matter locally but will be important for potential external consumers once the package is published)